api/v2/performanceprofile_types.go

```diff
@@ -123,6 +126,36 @@ type NUMA struct {
 	TopologyPolicy *string `json:"topologyPolicy,omitempty"`
 }
 
+// Net defines a set of network related features
+type Net struct {
+	// UserLevelNetworking defines if the specified network devices . Defaults to "false".
+	UserLevelNetworking *bool `json:"userLevelNetworking,omitempty"`
+	// Devices contains a list of network device representations that will be
+	// set with a netqueue count equals to CPU.Reserved .
+	// If no devices are specified then the default is all devices.
+	Devices []Device `json:"devices,omitempty"`
+}
+
+// Device defines a way to represent a network device in several options:
+// device name, vendor ID, model ID, PCI path and MAC address
+type Device struct {
+	// Network device name
+	// +optional
+	Name *string `json:"name,omitempty"`
```

Comment:
Should this be `Interface` or `InterfaceName`?

Reply:
It would probably make sense to change it to InterfaceName for clarity ,yes.